kernel/kernel-api/src/main/java/io/delta/kernel/internal/deletionvectors/DeletionVectorStoredBitmap.java

```diff
@@ -60,17 +60,24 @@ public RoaringBitmapArray load(FileSystemClient fileSystemClient) throws IOExcep
         } else { // isOnDisk
             String onDiskPath = dvDescriptor.getAbsolutePath(tableDataPath.get());
 
-            // TODO: this type is TBD
-            Tuple2<String, Tuple2<Integer, Integer>> dvToRead =
-                new Tuple2(
-                    onDiskPath, // filePath
-                    new Tuple2(
-                        dvDescriptor.getOffset().orElse(0), // offset
-                        // we pad 4 bytes in the front for the size
-                        // and 4 bytes at the end for CRC-32 checksum
-                        dvDescriptor.getSizeInBytes() + 8 // size
-                    )
-                );
+            FileReadRequest dvToRead = new FileReadRequest() {
```

Comment:
I wonder if there is some Java best practice we could apply here. Should we just make a class? We could easily pass in these 3 values to a constructor of a class that implements this interface.

Reply:
I think this is fine for now. If we end up having lots of this same anonymous class, then we can reconsider

Reply:
Agreed

Reply:
flyby comment, I dont have full context on how this is used. This is something that is probably can fixed later as well as we get some more feedback and clarity.